Eastern-Research-Group/csb-rebate-forms-app
# Feature/update prf field names

## Related Issues:
* CSBAPP-275

## Main Changes:
* Update org field names used in new 2023 PRF submission to match latest used in the Formio form definition.

## Steps To Test:
1. Ask the BAP to change an existing 2023 FRF submission's status to "Accepted"
2. Create a new 2023 PRF submission.
3. Ensure the injected data for organizations is correct and matches the field names used in the Formio form definition.
